app/components/certificate/request.tsx

```diff
@@ -0,0 +1,21 @@
+import { Flex, Heading, Text, HStack } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  cert: Certificate;
```

Comment:
Pass the specific thing down you want to show: `domainName`

Reply:
I am not quite sure what you mean by this?